README.md

```diff
@@ -173,6 +173,75 @@ loader
 
 If `options.resolve` is truthy, speccy will resolve _external_ references.
 
+### Adding Custom Rules
+
+You're able to create your own custom rules for Speccy. To accomplish this, follow these steps:
+
+1. Add new rule object to `rules/default.json`
```

Comment:
This is very cool advice, but might be better off in `CONTRIBUTING.md`, under the same section name you have here. The documentation I was looking for was more along the lines of "How to create your own rules files", which would show how to use existing rule actions, explaining how they reference objects, etc. Does that make sense?

Reply:
Yeah, I understand what you're saying. My thinking was documenting the process for PR #146. I'll make the corrections.